src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -77,12 +77,28 @@ status "Checking for UAA zone ${UAA_ZONE_ID}..."
 uaac target <%= uaac_ssl %> '<%= p("scf.uaa.root-zone.url") %>'
 uaac token client get default_zone_admin --secret '<%= p("uaa.admin.client_secret") %>'
 
-if uaac curl <%= curl_ssl %> "/identity-zones/${UAA_ZONE_ID}" | grep --quiet '404 Not Found' ; then
+if uaac curl <%= curl_ssl %> "/identity-zones/${UAA_ZONE_ID}" | grep --quiet '404' ; then
     status "Creating UAA zone ${UAA_ZONE_ID}"
     uaac curl <%= curl_ssl %> -X POST /identity-zones -H 'Content-Type: application/json' -d '{
         "id": "'"${UAA_ZONE_ID}"'",
         "subdomain": "'"${UAA_ZONE_ID}"'",
-        "name": "'"${UAA_ZONE_ID}"'"
+        "name": "'"${UAA_ZONE_ID}"'",
+        "config": {
+          "userConfig": {
+            "defaultGroups": [
```

Comment:
Hmm, this should probably be coming from the opinions or something (next to where the list of authorities lives).